### PR TITLE
EZP-32016: Redesign multi select

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-32016
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Screenshot from 2020-10-15 11-05-00
Screenshot from 2020-10-15 11-04-48

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/core/custom.dropdown.js [hunk 1]
```diff
@@ -102,6 +106,8 @@
                 }
             }
 
+
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/core/custom.dropdown.js:109]: unneccessary newlines

File: src/bundle/Resources/public/js/scripts/core/custom.dropdown.js [hunk 2]
```diff
@@ -172,9 +181,37 @@
...
+            const selectedItemsContainer = this.container.querySelector(SELECTOR_SELECTION_INFO);
+            const selectedItems = this.container.querySelectorAll(SELECTOR_SELECTED_ITEM_IN_LABEL);
+            const selectedItemsOverflow = this.container.querySelector(SELECTOR_OVERFLOW_ITEM_INFO);
+
+            if (selectedItemsOverflow) {
+                selectedItems.forEach(item => { item.hidden = false; })
+                selectedItems.forEach(item => {
+                    itemsWidth += item.offsetWidth;
+
+                    if (itemsWidth > (selectedItemsContainer.offsetWidth - 120)) {
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/core/custom.dropdown.js:200]: Why 120? Some comment, it's magic number at the moment

File: src/bundle/Resources/public/js/scripts/core/custom.dropdown.js [hunk 3]
```diff
@@ -172,9 +181,37 @@
...
+                        numberOfOverflowItems++;
+                        item.hidden = true;
+                    }
+                });
+
+                if (numberOfOverflowItems !== 0) {
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/core/custom.dropdown.js:206]: just `if (numberOfOverflowItems)`

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/js/scripts/core/custom.dropdown.js (hunk 2 above)
Comment (dew326) [reply src/bundle/Resources/public/js/scripts/core/custom.dropdown.js:200]: Comments in code are bad, assign this to meaningful variable :)

**Review by dew326 — APPROVED** [review]

**Pull request merged**